motoman_gp200r_support/urdf/gp200r_macro.xacro

```diff
@@ -0,0 +1,164 @@
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_1_s.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_2_l">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_2_l.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_2_l.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_3_u">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_3_u.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_3_u.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_4_r">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_4_r.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_4_r.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_5_b">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_5_b.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_5_b.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}link_6_t">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_6_t.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/link_6_t.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+    </link>
+    <link name="${prefix}tool0"/>
```

Comment:
Please move `tool0`'s `link` to the end of the file together with its `joint` declaration block (place it after the `base` `link` and `joint`).

Refer to [sia30d_macro.xacro](https://github.com/ros-industrial/motoman/blob/21c83deee272215fc307874ec32447fcffabdbee/motoman_sia30d_support/urdf/sia30d_macro.xacro#L165-L179) for an example.

Reply:
I moved the tool0 link.